main.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/gorilla/mux"
+)
+
+// Message object
+type msg struct {
+	Message string

```

Comment:
Make the json representation of this attribute by adding 
```
`json:""`
```
 at the end. It should show `"message"` (lowercase) in responses.